src/url-replacements.js

```diff
@@ -95,6 +95,17 @@ class UrlReplacements {
       return documentInfoFor(this.win_).pageViewId;
     });
 
+    this.set_('QUERY_STRING', (param, defaultValue = "") => {
```

Comment:
Ah, I thought you were looking for the entire query string. I'd rather call this `QUERY_PARAM`